mod/network.php

```diff
@@ -499,19 +499,24 @@ function networkThreadedView(App $a, $update, $parent)
 		$o .= status_editor($a, $x);
 	}
 
-	$sql_table = $update ? '`item`' : '`thread`';
+	$condition1 = ['uid' => local_user()];
+	$condition2 = [];
```

Comment:
Please use meaningful names for the condition variables. What's the difference between them since they're merged together on line 618?

Reply:
One condition is a simple associate array, the other one is for some more complicate stuff. I just wanted to avoid calling the merge command for every call.

Reply:
In this case I suggest:
- `$condition1` => `$conditionFields`
- `$condition2` => `$conditionStrings`

Reply:
Okay, sounds good.